docs/docusaurus/docs/changelog.md

```diff
@@ -10,6 +10,25 @@ title: Changelog
 - Deprecation warnings are accompanied by a moniker (as a code comment) indicating when they were deprecated.  For example: `# deprecated-v0.13`
 - Changes to methods and parameters due to deprecation are also noted in the relevant docstrings.
 
+### 0.18.2
+* [DOCS] Update Connect to SQL Database Data Assets ([#8918](https://github.com/great-expectations/great_expectations/pull/8918))
+* [DOCS] DSB-353: Add GX Cloud Architecture and Deployment Patterns page ([#8904](https://github.com/great-expectations/great_expectations/pull/8904))
+* [DOCS] Update descs for custom expectations ([#8923](https://github.com/great-expectations/great_expectations/pull/8923))
+* [DOCS] Update cloud docs for data assets ([#8932](https://github.com/great-expectations/great_expectations/pull/8932)) (thanks @r34ctor)
```

Comment:
Looks like this is Nevin on Devrel: https://github.com/r34ctor, should be removed (should add him to the contributors list)
```suggestion
* [DOCS] Update cloud docs for data assets ([#8932](https://github.com/great-expectations/great_expectations/pull/8932))
```

Reply:
Contributors list: https://github.com/great-expectations/great_expectations/blob/develop/.github/teams.yml

Reply:
Updated: https://github.com/great-expectations/great_expectations/pull/8955